Sources/SWBCore/Settings/Settings.swift

```diff
@@ -1573,6 +1573,12 @@ private class SettingsBuilder {
             addPlaygroundSettings(specLookupContext.platform)
         }
 
+        if let project, project.isPackage {
```

Comment:
We should restrict this to Apple platforms, since ARC is not available on Windows/Linux/etc. I think it would also make sense to push it somewhere user-supplied configuration is still able to override it - maybe in `addPlatformSettings` or similar